python/ql/src/meta/analysis-quality/PointsToResolvableCalls.ql

```diff
@@ -0,0 +1,14 @@
+/**
+ * @name Resolvable calls by points-to
+ * @description The number of (relevant) calls that could be resolved to its target.
```

Comment:
The description in `PointsToResolvableCallRatio.qll` uses `can`, but this one uses `could`. We should probably stick to a single tense and use it everywhere. :slightly_smiling_face: 
```suggestion
 * @description The number of (relevant) calls that can be resolved to a target.
```

Reply:
... Which I then promptly forget to do in the rest of my review.